src/index.js

```diff
@@ -156,7 +152,7 @@ function hyphenate (str) {
 }
 
 function addPx (prop, value) {
-  if (typeof value !== 'number' || unitlessProps[prop]) return value
+  if (typeof value !== 'number' || unitlessProps[prop] !== void 0) return value
```

Comment:
imo `void 0` should be `undefined` as it's easier to understand what the condition is for. Minifiers will change it to void anyway, so no need to save space manually.